src/libraries/System.Text.Json/src/System/Text/Json/Serialization/JsonSerializerDefaults.cs

```diff
@@ -0,0 +1,21 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.Text.Json
+{
+    /// <summary>
+    /// Signifies what default options are instantiated in <see cref="JsonSerializerOptions"/>.
+    /// </summary>
+    public enum JsonSerializerDefaults : byte
+    {
+        /// <summary>
+        /// General-purpose defaults
+        /// </summary>
+        General = 0,
+        /// <summary>
+        /// Web scenarios defaults
+        /// </summary>
```

Comment:
/// <summary>
        /// Specifies that values should be used more appropriate to web-based scenarios.
        /// </summary>
        /// <remarks>
        /// This option implies that property names are treated as case-insensitive and that "camelCase" name formatting should be employed.
        /// </remarks>